python-threatexchange/threatexchange/cli/cli_config.py

```diff
@@ -172,6 +172,48 @@ def delete_collab(self, collab: collab_config.CollaborationConfigBase) -> None:
         self.path_for_collab_config(collab).unlink(missing_ok=True)
 
 
+@dataclass
+class _SignalExchangeAccessor:
+    """Convenience wrapper for operations on the SignalExchangeAPI"""
+
+    _parent: "CLISettings"
```

Comment:
question unrelated to review: is there a name for this pattern python? can you do something like this without needed the quotes and is mypy smart enough to figure out this is a type/class when used like this?

Reply:
This is a forward reference: https://peps.python.org/pep-0484/#forward-references